flowzone.yml

```diff
@@ -3061,7 +3094,7 @@ jobs:
       - *getGitHubAppToken
       - *checkoutVersionedSha
 
-      - name: Set up toolchain ${{ matrix.target }}
```

Comment:
Any idea why it removed this?

Reply:
because `${{ matrix.target }}` is not defined in this context/job.